spec/android_spec.rb

```diff
@@ -209,7 +209,7 @@
 wlan0    UP                                     0.0.0.0/0   0x00001003 fc:c2:de:6a:04:9e
 EOF
           device = DeviceAPI::Android::Device.new(serial: 'SH34RW905290')
-          allow(Open3).to receive(:capture3) { [out, '', $STATUS_ZERO] }
+          allow(Open3).to receive(:capture3) { [out, '', STATUS_OK] }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.